setup.py

```diff
@@ -38,7 +38,7 @@
         'GitPython==3.0.8',
         'boltons==19.1.0',
         'dnspython==1.16.0',
-        'evergreen.py==0.6.14',
+        'evergreen.py==1.0.2',
```

Comment:
fixes error `(data_string, format))\nValueError: time data '2020-02-19T15:27:49Z' does not match format '%Y-%m-%dT%H:%M:%S.%fZ'"`

Reply:
:+1: